include/wlr/types/wlr_relative_pointer_v1.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_RELATIVE_POINTER_V1_H
+#define WLR_TYPES_WLR_RELATIVE_POINTER_V1_H
+
+#include <wayland-server.h>
+
+
+/**
+ * This protocol specifies a set of interfaces used for making clients able to
+ * receive relative pointer events not obstructed by barriers (such as the
+ * monitor edge or other pointer barriers).
+ */
+
+
+/**
+ * A global interface used for getting the relative pointer object for a given
+ * pointer.
+ */
+
+struct wlr_relative_pointer_manager_v1 {
+	struct wl_global *global;
+	struct wl_list resources;
+	struct wl_list relative_pointers;
```

Comment:
This should be a list of `wlr_relative_pointer_v1`, not a list of resources.

Reply:
Can you add a little comment `// wlr_relative_pointer_v1::link` to make it clear it's a list of these?

Reply:
Bump

Reply:
Bump